DotBot.py

```diff
@@ -173,6 +171,10 @@ def _transmit(self):
             sender      = self,
         )
 
+        # schedule re-transmit
+        self.simEngine.schedule( self.simEngine.currentTime() + 1, self._transmit,
```

Comment:
The `+1` here is a magic value. What you want is to retransmit at the same period as the Orchestrator's `COMM_DOWNSTREAM_PERIOD_S`, so use that.